Source/WebCore/platform/mediastream/gstreamer/GStreamerVideoCapturer.cpp

```diff
@@ -139,7 +138,6 @@ GstElementFactory* GStreamerVideoCapturer::getParser (const char* format)
     parser_factory = GST_ELEMENT_FACTORY (g_list_first(parsers)->data);
```

Comment:
Same changes as suggested for `getEncoder()`.

Reply:
Added changes.